src/data/markdown/docs/01 guides/05 Testing Guides/04 Running large tests.md

```diff
@@ -485,8 +478,65 @@ Results
+k6 run --execution-segment "0:1/2"   --execution-segment-sequence "0,1/2,1" my-script.js
+k6 run --execution-segment "1/2:1"   --execution-segment-sequence "0,1/2,1" my-script.js
+```
+
+```bash
+## split the load of my-script.js across three machines
+k6 run --execution-segment "0:1/3"   --execution-segment-sequence "0,1/3,2/3,1" my-script.js
+k6 run --execution-segment "1/3:2/3" --execution-segment-sequence "0,1/3,2/3,1" my-script.js
+k6 run --execution-segment "2/3:1"   --execution-segment-sequence "0,1/3,2/3,1" my-script.js
+```
+
+```bash
+## split the load of my-script.js across four machines
+k6 run --execution-segment "0:1/4"     --execution-segment-sequence "0,1/4,2/4,3/4,1" my-script.js
+k6 run --execution-segment "1/4:2/4"   --execution-segment-sequence "0,1/4,2/4,3/4,1" my-script.js
+k6 run --execution-segment "2/4:3/4"   --execution-segment-sequence "0,1/4,2/4,3/4,1" my-script.js
+k6 run --execution-segment "3/4:1"     --execution-segment-sequence "0,1/4,2/4,3/4,1" my-script.js
+```
+
+</div>
+
+However - at this moment - the distributed execution mode of k6 is not entirely functional. The current limitations are:
+
+- k6 does not provide a `test coordinator` or `master instance` to coordinate the distributed execution of the test. Alternatively, you can use the [k6 REST API](/misc/k6-rest-api) and [--paused](/using-k6/options#paused) to synchronize the multiple k6 instances' execution. 
+- Each k6 instance evaluates [Thresholds](/using-k6/thresholds) independently - excluding the results of the other k6 instances. If you want to disable the threshold execution, use [--no-thresholds](/using-k6/options#no-thresholds).
+- k6 reports individually the metrics for each instance. Depending on how you store the load test results, you'll have to aggregate some metrics to calculate them correctly. 
+
+
+> The k6 goal is to support a native open-source solution for distributed execution. If you want to follow the progress, subscribe to the [distributed execution issue](https://github.com/loadimpact/k6/issues/140) on GitHub.
+
+
+## Large-scale tests in k6 Cloud
+
+Building a load testing infrastructure to support running large-scale distributed tests is a challenging engineering project.
+
+[k6 Cloud](https://k6.io/cloud) - our commercial offering - provides an instant solution for running this type of testing. Amongst other [benefits](https://k6.io/docs/cloud#how-can-it-help-me), our cloud platform will prevent your engineering team from building and maintaining the software and infrastructure of your load testing solution.
+
+Rolling your own or buying a load testing solution is a decision to consider that depends on your project, the type of testing, your team's expertise, organization's aspects, etc. If you doubt which solution is a better fit for your project, reach us on the [Community Forum](https://community.k6.io/) or to the [Cloud Support team](https://k6.io/contact) to help you with your questions. 
```

Comment:
I would change `If you doubt which solution is` to `If you aren't sure about which solution is`

Reply:
https://github.com/loadimpact/k6-docs/pull/102/commits/d602a8d75d00320b4adc4a4f5b5e4b1c77e2daa5